test-suite/Makefile

```diff
@@ -46,7 +46,7 @@ else
   export FINDLIB_SEP=:
 endif
 
-# Be careful to use COQPREFIX when in local mode
+# Be careful to use COQPREFIX when prefix is not set
 ifeq ($(COQPREFIX),local)
```

Comment:
When is prefix not set?

Reply:
There should be no case anymore, however I am having problems following configure's logic as it is a bit convoluted [working on a rewite], and indeed there is the check in configure that should set it.

Additionally, it is not clear to me what should we do when prefix is not set by the user.